recipes/folly/all/conanfile.py

```diff
@@ -66,12 +67,13 @@ def requirements(self):
         self.requires("gflags/2.2.2")
         self.requires("glog/0.4.0")
         self.requires("libevent/2.1.12")
-        self.requires("openssl/1.1.1m")
+        self.requires("openssl/1.1.1n")
         self.requires("lz4/1.9.3")
         self.requires("snappy/1.1.8")
         self.requires("zlib/1.2.11")
         self.requires("zstd/1.4.9")
-        self.requires("libdwarf/20191104")
+        if not is_msvc(self):
+            self.requires("libdwarf/20191104")
```

Comment:
Why?

Reply:
visual studio does not use DWARF format, so there is no point in dragging this dependency (used only in exception_tracer and symbolizer), especially considering the headers of libelf (requirement of libdwarf) conflict with msvc : https://github.com/conan-io/conan-center-index/pull/10021

Reply:
> especially considering the headers of libelf (requirement of libdwarf) conflict with msvc : https://github.com/conan-io/conan-center-index/pull/10021

From what I understand, it conflicts with Windows SDK, so it should be removed if windows, not only msvc.